dist/index.js

```diff
@@ -45,6 +56,12 @@ var getTagLengthBytes = (tag) => {
     case 25:
     case 29:
       return 2;
+    case 37:
```

Comment:
can these be combined with case 15, case 25, case 29 above? (as they all return 2)?